### PR TITLE
view list of books, create a filter pipe

### PR DESCRIPTION
**Pull request by kalzipher**
## Summary 
- Create pipe filter
- Create list books
- Create book component

## Screenshot
image

## Trello card
https://trello.com/c/mpdOjOSR/6-book-list-auth-header

### CODE REVIEW

File: src/app/guards/unauth/unauth.guard.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ export class UnauthGuard implements CanActivate {
   }
 
   private isAuth(): boolean {
+    console.log(localStorage.getItem(KEYS_STORAGE.access_token))
```
Comment (betoobernal) [inline src/app/guards/unauth/unauth.guard.ts]: remove log

File: src/app/screens/books/components/book/book.component.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class="book">
+  <div class="container-book-poster">
+    <img class="book-poster">
+  </div>
+  <h3 style="display: block;">titulo del libro</h3>
```
Comment (betoobernal) [inline src/app/screens/books/components/book/book.component.html]: use an external file for css

File: src/app/screens/books/components/book/book.component.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.book {
+  box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+  max-width: 300px;
+  min-width: 200px;
+  padding: 1rem;
+  position: relative;
+  transition: all 0.3s cubic-bezier(.25,.8,.25,1);
```
Comment (betoobernal) [inline src/app/screens/books/components/book/book.component.scss:7]: is all value necessary?

Comment (kalzipher) [reply src/app/screens/books/components/book/book.component.scss:7]: I use "all" to make the transition more flexible apply to all properties.

Do you think that is necessary to apply to an exact property?

**Review by betoobernal — APPROVED** [review]

**Pull request merged**